database/factories/ModelFactory.php

```diff
@@ -95,3 +96,17 @@
 $factory->defineAs(User::class, 'admin', function () use ($factory) {
     return array_merge($factory->raw(User::class), ['role' => 'admin']);
 });
+
+// Tag Factory
+$factory->define(Tag::class, function (Generator $faker) {
+    $tagName = str_random(10);
+
+    return [
+        'tag_name' => $tagName,
+        'tag_slug' => str_slug($tagName, '-'),
+    ];
+});
+
+$factory->defineAs(Tag::class, 'Hide In Gallery', function () use ($factory) {
```

Comment:
I tried to pass a variable `$tagName` instead of 'Hide In Gallery` here but it didn't like that!

Reply:
`$tagName` is scoped to the callback function in the statement before, which is why you probably can not use it.

Reply:
ah you're totally right, thanks @sbsmith86 !